build.gradle

```diff
@@ -61,16 +69,46 @@ repositories {
     mavenCentral()
 }
 
+// Write the plugin's classpath to a file to share with the tests
+task createClasspathManifest {
+  def outputDir = file("$buildDir/$name")
+
+  inputs.files sourceSets.main.runtimeClasspath
+  outputs.dir outputDir
+
+  doLast {
+    outputDir.mkdirs()
+    file("$outputDir/plugin-classpath.txt").text = sourceSets.main.runtimeClasspath.join("\n")
+  }
+}
+
+tasks.test.dependsOn tasks.createClasspathManifest
+if (findProject(':vendor') != null) {
```

Comment:
I was also confused and I found the problem which I'm still debugging. On Mac OS X and probably Linux, your suggestion worked. However, on Windows, it doesn't seems to work. It fail with "Task with path 'vendor:protobufExecutable' not found in root project 'protobuf-gradle-plugin'". I will try to figure out why this happen as I prefer your solution by far!